### PR TITLE
US-002 + US006 job specification page with responsibilities for each role

### PR DESCRIPTION
**Pull request by RyanCunning1**
Job specification page with links to sharepoint, role names and description of responsibilities 
updated with removal of whitespace

### CODE REVIEW

File: Views/UITests/src/main/java/pages/RolesPage.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    @FindBy(xpath = "//*[@id=\"JobSpecTable\"]/tbody/tr[1]/th[1]") private WebElement tableRoleName;
+    @FindBy(xpath = "//*[@id=\"JobSpecTable\"]/tbody/tr[1]/th[2]") private WebElement tableBandTier;
+    @FindBy(xpath = "//*[@id=\"JobSpecTable\"]/tbody/tr[1]/th[3]") private WebElement tableResponsibility;
+    @FindBy(xpath = "//*[@id=\"JobSpecTable\"]/tbody/tr[1]/th[4]") private WebElement tableJobSpecURL;
+
+
+    public RolesPage(WebDriver driver) {
+        super(driver);
+    }
+
```
Comment (jkelso662) [inline Views/UITests/src/main/java/pages/RolesPage.java:20]: Remove double and triple spacing.

File: Views/UITests/src/main/java/pages/RolesPage.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    public String tableResponsibilityCheck(){
+        return tableResponsibility.getText();
+    }
+
+    public String tableJobSpecURLCheck(){
+        return tableJobSpecURL.getText();
+    }
+
```
Comment (jkelso662) [inline Views/UITests/src/main/java/pages/RolesPage.java:42]: Too many blank lines!

**Review by jkelso662 — CHANGES_REQUESTED** [review]
Mostly white space again. Sorry!

**Review by jkelso662 — APPROVED** [review]
Much better. Typically you would see 0 blank lines in an HTML file but this might not be as strongly forced in some teams.

File: application/apiService/src/test/java/com/kainos/ea/apiService/controllers/apiServiceTest.java [hunk 3]
```diff
@@ -77,6 +78,73 @@ void mapResultSet() throws SQLException, JSONException {
...
+
+        apiService.getConnection();
+        String InvalidRoleName = "Invalid";
+        String expectedOutput = "404 NOT_FOUND Not Found";
+
+        ResponseEntity<String> apiResponse = apiService.getRoleSpec(InvalidRoleName);
+        String apiResponseString = apiResponse.toString();
+
+        assertTrue(apiResponseString.contains(expectedOutput));
+
```
Comment (liammc98) [inline application/apiService/src/test/java/com/kainos/ea/apiService/controllers/apiServiceTest.java:118]: Whitespace here, please remove

**Review by liammc98 — APPROVED** [review]
Looks like all requested changes were done

**Pull request merged**